src/components/interface/AudioPlay.jsx

```diff
@@ -1,26 +1,82 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/core/styles';
+
 import PlayCircleOutlineIcon from '@material-ui/icons/PlayCircleOutline';
 import styles from '../styles';
+import Button from '../interface/Button';
+import * as firebase from 'firebase';
 
 class AudioPlayer extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      disabled: false
+      audioFile: undefined,
+      metadataLoaded: false
     };
   }
 
-  // TODO: handle audio playing file
+  loadAudio = () => {
+    this.pathRef.getDownloadURL().then(function (url) {
```

Comment:
Javascript style is that we use arrow functions instead of `function (...) { ... }`.

`.then( (url) => { ... } )`